readthedocs/config/config.py

```diff
@@ -245,7 +245,11 @@ def validate(self):
     @property
     def python_interpreter(self):
         ver = self.python_full_version
-        return 'python{}'.format(ver)
+        if not ver or isinstance(ver, (int, float)):
+            return 'python{}'.format(ver)
+
+        # Allow to specify ``pypy3.5`` as Python interpreter
+        return ver
```

Comment:
Just a note that we can do this better later https://github.com/rtfd/readthedocs.org/issues/4343